src/coprocessor/dag/expr/compare.rs

```diff
@@ -216,6 +344,27 @@ impl FnCall {
         let escape = try_opt!(self.children[2].eval_int(ctx, row)) as u32;
         Ok(Some(like(&target, &pattern, escape, 0)? as i64))
     }
+
+    pub fn regexp(&self, ctx: &mut EvalContext, row: &[Datum]) -> Result<Option<i64>> {
```

Comment:
Please create a new mod named `builtin_like` and moves the related functions into it.